feat: setting type of node

aWN4Y25pa2EK opened the pull request:
1. Enables `UDP` 
2. Adds `arabica-11` example

smuu reviewed (dismissed):
LGTM

smuu commented on charts/celestia-node/values.yaml:
```diff
@@ -74,6 +74,12 @@ diagnosticMode:
 ##
 node:
+  ## Select from: bridge, full, light
+  ## Valid values:
+  ## - bridge
+  ## - full
+  ## - light:
```
We need to add some validation to ensure that the user sets one of these values.
One idea would be to use this: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/templates/_helpers.tpl#L46

aWN4Y25pa2EK replied:
great idea!

smuu commented on charts/celestia-node/templates/statefulset.yaml:
```diff
@@ -207,16 +207,22 @@ spec:
                   AUTH_TOKEN=$(celestia bridge auth read --node.store=/home/celestia 2>/dev/null);
+                  {{- if eq .Values.node.nodeType "bridge" }}
+                  curl --max-time 0.5 -s -X POST -H "Content-type: application/json" -H "Accept: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d '{"id":1,"jsonrpc":"2.0","method":"header.SyncWait","params":[]}' "http://localhost:26658";
+                  {{- else if eq .Values.node.nodeType "full" }}
```
`LN` has the same probe as `FN`, so we could add it here

aWN4Y25pa2EK replied:
noted!

smuu commented on charts/celestia-node/values.yaml:
```diff
@@ -74,6 +74,12 @@ diagnosticMode:
+  ## - bridge
+  ## - full
+  ## - light:
+  nodeType: bridge
```
Can we move this to `node.settings.nodeType` as we have all the other values to specify what we run?

smuu replied:
This should also change the args here: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/values.yaml#L511

This way, we can start different node types without touching the `args` in the values file.

aWN4Y25pa2EK commented:
- moved `nodeType` to `node.settings` 
- added logic validation for `nodeType` deployment
- added `readinessprobe` to be the same as the one used by the `FN` 
- added anchors to simplify the `values.fies`
